### PR TITLE
rtic-monotonics: Add documentation for i.MX RT and other doc fixes

### PR DESCRIPTION
**Pull request by Finomnis**

### CODE REVIEW

File: rtic-monotonics/src/systick.rs [hunk 1]
```diff
@@ -5,7 +5,7 @@
 //!
 //! | Feature          | Tick rate | Precision |
 //! |:----------------:|----------:|----------:|
-//! | (none / default) |  1 Hz     |      1 ms |
+//! | (none / default) |  1 Hz     |       1 s |
```
Comment (korken89) [inline rtic-monotonics/src/systick.rs]: This should say 1 kHz, 1 ms.

**Review by korken89 — CHANGES_REQUESTED** [review]
Thanks for the PR! There is a small error that needs fixing, after that I'll merge :D

File: rtic-monotonics/src/systick.rs (hunk 1 above)
Comment (Finomnis) [reply rtic-monotonics/src/systick.rs]: My bad, I corrected it in the wrong direction.

**Comment by Finomnis** [comment]
@korken89 fixed :)

**Review by korken89 — APPROVED** [review]

**Comment by Finomnis** [comment]
@korken89 Any rough idea on when 1.3.0 will be released to crates.io?

**Pull request merged**